src/config/firebaseConfig.js

```diff
@@ -30,7 +30,13 @@ firebase.initializeApp(config);
 
 // firebase utils
 const db = firebase.firestore();
-const loginProvider = new firebase.auth.GoogleAuthProvider();
+const loginProviderGoogle = new firebase.auth.GoogleAuthProvider();
+const loginProviderMS = new firebase.auth.OAuthProvider('microsoft.com');
+loginProviderMS.setCustomParameters({
+  // "tenant" parameter in case you are using an Azure AD tenant.
+  // The default value is "common".
+  tenant: process.env.VUE_APP_MICROSOFT_TENANT
```

Comment:
Hvis `VUE_APP_MICROSOFT_TENANT` er undefined så vil den sende det inn som undefined. Vil da provideren skjønne at det skal settes default til `common` ? Hvis ikke så hadde jeg kanskje anbefalt å legge inn `process.env.VUE_APP_MICROSOFT_TENANT || 'common'` så den defaulter til common hvis den ikke finner noe i env

Reply:
Veldig enig. Fikser den.

Reply:
Da kan også selve env være optional, så det er nice for de som ikke bruker microsoft (jeg har planer om å flytte mye av koden din inn til okr-trackeren vår / lager en PR inn i vårt repo, så dette er bare gull)